samples/snippets/csharp/VS_Snippets_Wpf/WindowClosingSnippets/CSharp/DataWindow.xaml.cs

```diff
@@ -1,4 +1,3 @@
-//<SnippetWindowClosingCODEBEHIND1>
```

Comment:
The four changes involving **SnippetWindowClosingCODEBEHIND***x* have broken the build. The code snippets are used in one more place: [Window.Closing](https://docs.microsoft.com/dotnet/api/system.windows.window.closing).